plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaFilterManager.java

```diff
@@ -129,11 +129,15 @@ public KafkaFilteringResult getKafkaFilterResult(
             if (offsetTimestampRanged.isPresent()) {
                 try (KafkaConsumer<byte[], byte[]> kafkaConsumer = consumerFactory.create(session)) {
                     Optional<Range> finalOffsetTimestampRanged = offsetTimestampRanged;
-                    partitionBeginOffsets = overridePartitionBeginOffsets(partitionBeginOffsets,
-                            partition -> findOffsetsForTimestampGreaterOrEqual(kafkaConsumer, partition, finalOffsetTimestampRanged.get().getBegin()));
+                    if (offsetTimestampRanged.get().getBegin() > INVALID_KAFKA_RANGE_INDEX) {
```

Comment:
It would be better to leave comment why we ignore instead of an exception.

Reply:
OK, I added the reason why we need.